socrata/socrata_upload.py

```diff
@@ -10,6 +10,14 @@
 from pyathena import connect
 from pyathena.pandas.cursor import PandasCursor
 
+# Create a session object so HTTP requests can be pooled
+s = requests.Session()
+s.verify = True
+s.auth = (
+    str(os.getenv("SOCRATA_USERNAME")),
+    str(os.getenv("SOCRATA_PASSWORD")),
+)
```

Comment:
**[Question, non-blocking]** I see that we were doing this before, so I get that this auth practice is already on the main branch, but now that my memory is refreshed it's making me wonder -- is there a way to auth to the API using only a token instead of also needing to set our account username/password? Without understanding the details, it strikes me as a super weird pattern (the token should be able to perform auth on its own), and I'm a bit nervous reading and sending username/password given that a leaked username/password could cause more damage than a leaked token.